woodwork/schema.py

```diff
@@ -377,30 +377,42 @@ def _set_index_tags(self, index):
     def _set_time_index_tags(self, time_index):
         self.columns[time_index]['semantic_tags'].add('time_index')
 
-    def _filter_cols(self, include, col_names=False):
+    def _filter_cols(self, include=None, exclude=None, col_names=False):
         """Return list of columns filtered with any of: semantic tags, LogicalTypes, column names
 
         Args:
             include (str or LogicalType or list[str or LogicalType]): parameter or list of parameters to
-                filter columns by. Can be Logical Types or Semantic Tags.
+                filter columns by. Can be Logical Types or Semantic Tags. Columns that match will be
+                included in the returned list of columns.
+            exclude (str or LogicalType or list[str or LogicalType]): parameter or list of parameters to
+                filter columns by. Can be Logical Types or Semantic Tags. Columns that match will be
+                excluded from the returned list of columns.
 
             col_names (bool): Specifies whether to filter columns by name. Defaults to False.
 
         Returns:
             List[str] of column names that fit into filter.
         """
-        if not isinstance(include, list):
+        assert not (include and exclude), "Cannot specify both include and exclude"
```

Comment:
I like the assertion here that they're not both present. Would make sense to also have the assertion that they're not both absent?

Reply:
My reasoning for putting this assertion here is help guard against an unintended behavior and to make sure the logic that follows is valid. If both are specified, there will not be any errors raised without this assertion, so the assertion is really a protection for us as developers to highlight we did something wrong.

If both are `None` an error will result - albeit not a helpful error, but since it is a private method I didn't think that was a big deal.

With all that, I was thinking the second assertion wasn't needed, but there is no harm in adding it (aside from one more line of code) if you think it would be helpful.

Reply:
ahh I see--I think that raising an error when there would otherwise be none is a good reason for the Assertion, and I see your point of not adding the second one if it'll raise an error on its own. Fine not adding!